server/services/interventionsService.ts

```diff
@@ -95,6 +104,33 @@ export default class InterventionsService {
     return sanitisedError
   }
 
+  serializeDeliusServiceUser(deliusServiceUser: DeliusServiceUser | null): ServiceUser {
+    if (!deliusServiceUser) {
+      return {} as ServiceUser
+    }
+
+    const currentDisabilities = deliusServiceUser.disabilities
```

Comment:
just a note that we discussed including "ended" disabilities with a postfix such as `(expired)`, but having read this i do think it's probably better to just not include them at all.

Reply:
Yeah, I initially did put this in, but this morning had the thought that actually it's probably not relevant to the service provider if it's not a current disability.